modules/mobile/app/controllers/mobile/application_controller.rb

```diff
@@ -32,8 +22,12 @@ def authenticate
       @current_user
     end
 
+    def sis_authentication?
+      request.headers['Sign-In-Method'] == 'sis'
```

Comment:
Anyone have better ideas for a header name?

Reply:
def check_sign_in_header?

Reply:
@kpethtel

Reply:
I think this is a miscommunication. I was asking about how we should structure the header name/value. As coded, it would be:
`Sign-In-Method : sis`

This was just made up during the coding process, so I'm curious if anyone has opinions.

Reply:
`authentication-method` works for me. I also considered a boolean. It was a bit of a coin flip.

Reply:
Yeah, boolean or not doesn't really matter too much to me